AmplifyPlugins/Predictions/AWSPredictionsPlugin/Service/Predictions/AWSPredictionsService+Transcribe.swift

```diff
@@ -31,6 +31,17 @@ extension AWSPredictionsService: AWSTranscribeStreamingServiceBehavior {
         request.mediaSampleRateHertz = 8_000
 
         transcribeClientDelegate.connectionStatusCallback = { status, error in
+            guard status != .closed && error == nil else {
```

Comment:
Before, the error returned on line 33 is never used, we should not skip error checking.

Reply:
```
if status == .closed && error != nil {
  
} else is status == .connected {

}
```